### PR TITLE
ref(releases): Added tests to all release serializers.

### PR DESCRIPTION
**Pull request by lauryndbrown**
Adding these test as I'm preparing to unify the various `ReleaseSerializer`s in a central location. These tests are to hopefully ensure that no significant changes are made to the serializers.

### CODE REVIEW

**Comment by lauryndbrown** [comment]
Please let me know if you see any weaknesses in my tests or things I might have missed that will be useful when refactoring. :)

File: tests/sentry/api/endpoints/test_organization_releases.py [hunk 1]
```diff
@@ -1343,3 +1346,156 @@ def test_commits_with_patch_set(self):
         self.assert_file_change(file_changes[1], 'A', 'templates/goodbye.html', commits[1].id)
         self.assert_file_change(file_changes[2], 'M', 'templates/hello.html', commits[1].id)
         self.assert_file_change(file_changes[3], 'D', 'templates/hola.html', commits[0].id)
+
+
+class ReleaseSerializerWithProjectsTest(TestCase):
```
Comment (markstory) [inline tests/sentry/api/endpoints/test_organization_releases.py]: Do you need database support in these tests? If not you can inherit from `unittest.TestCase` and get much faster execution time.

Comment (lauryndbrown) [reply tests/sentry/api/endpoints/test_organization_releases.py]: Hmm... by database support does that include lookup?  I'll play with it for a bit. I know when the serializer has a`UserField` it gets the user from the database based on the username. That's not all of them though. Let me try it and see what happens.

**Comment by lauryndbrown** [comment]
@markstory added `unittest.TestCase` where possible. Let me know what you think now.

**Review by markstory — APPROVED** [review]

**Pull request merged**